tools/ci_build/build.py

```diff
@@ -1312,140 +1312,6 @@ def run_onnxruntime_tests(args, source_dir, ctest_path, build_dir, configs):
                         cwd=cwd, dll_path=dll_path)
 
 
-def tensorrt_run_onnx_tests(args, build_dir, configs, onnx_test_data_dir,
-                            provider, num_parallel_models,
```

Comment:
The deleted tests are already covered in onnxruntime_test_all.

Reply:
i missed the other PR.
so if i'm understanding correctly, onnxruntime_test_all (which used to be only running unit tests)
are now running model tests?
But model tests are contained in an external zip package right? 
and now what is the role of onnx_test_runner?

Reply:
The test models are preloaded in our CI build machines, because they're large. 
onnx_test_runner and onnxruntime_test_all share the most of code, it is just onnx_test_runner won't be used in CI anymore, but people can still use it for debugging model issue, because it is more convenient.